tests/ltp/patches/setsid01.patch

```diff
@@ -0,0 +1,260 @@
+diff --git a/testcases/kernel/syscalls/setsid/setsid01.c b/testcases/kernel/syscalls/setsid/setsid01.c
```

Comment:
Please add description for the change. Look at other PRs for sample

Reply:
Done